src/tools/run-make-support/src/lib.rs

```diff
@@ -267,6 +267,13 @@ macro_rules! impl_common_helpers {
                 self.cmd.current_dir(path);
                 self
             }
+
+            /// Specify the edition year.
+            pub fn edition(&mut self, edition: &str) -> &mut Self {
+                self.cmd.arg("--edition");
+                self.cmd.arg(edition);
+                self
+            }
```

Comment:
This shouldn't be on `impl_common_headers` because that macro is designed to be usable for arbitrary command wrappers, so this should probably be on `Rustc` and `Rustdoc`. Some code duplication there is okay imo.

Reply:
Oh I see. Sounds good to me!